testing/connectors.py

```diff
@@ -151,7 +151,7 @@ def create_mocked_redis(*args: Any, **kwargs: Any) -> MockStrictRedis:
 @pytest.fixture(scope='session')
 def margo_connector() -> Generator[Connector[Any], None, None]:
     """MargoConnector fixture."""
-    host = '127.0.0.1'
+    # host = '127.0.0.1'
```

Comment:
We should either pin the host in tests or delete these commented out line. I'm not sure if there is an argument to be made for pinning the host so maybe we just delete these.